frontend/src/app/applications/fields/org-type.component.html

```diff
@@ -0,0 +1,48 @@
+<ng-container [formGroup]="applicantInfo" *ngIf="type === 'simple'">
+  <label for="applicant-type">Are you applying as an individual or organization?</label>
+  <ul class="usa-unstyled-list">
+    <li>
+      <input id="individual" type="radio" name="orgType" value="Person" formControlName="orgType">
+      <label for="individual">Individual</label>
+    </li>
+    <li>
+      <input id="organization" type="radio" name="orgType" value="Corporation" formControlName="orgType">
+      <label for="organization">Organization</label>
+    </li>
+  </ul>
+</ng-container>
+
+<ng-container [formGroup]="applicantInfo" *ngIf="type === 'complex'">
+  <label for="applicant-type">As an applicant, are you:</label>
+  <ul class="usa-unstyled-list">
+    <li>
+      <input id="individual" type="radio" name="orgType" value="individual" formControlName="orgType">
+      <label for="individual">Individual</label>
+    </li>
+    <li>
+      <input id="organization" type="radio" name="orgType" value="corporation" formControlName="orgType">
+      <label for="organization">Corporation</label>
+    </li>
+    <li>
+      <input id="llc" type="radio" name="orgType" value="llc" formControlName="orgType">
+      <label for="llc">Limited Liability Company</label>
+    </li>
+    <li>
+      <input id="llc" type="radio" name="orgType" value="partnership" formControlName="orgType">
```

Comment:
it looks like this  and all of the following `orgTypes` are still using llc. Please update.